expression/builtin_arithmetic_test.go

```diff
@@ -360,6 +360,14 @@ func (s *testEvaluatorSuite) TestArithmeticDivide(c *C) {
 			args:   []interface{}{nil, nil},
 			expect: nil,
 		},
+		{
+			args:   []interface{}{uint64(3), -3},
```

Comment:
@Tjianke 
It seems not working as expected. `err` returned by `evalBuiltinFunc` in line 389 should not be `IsNil` when overflow.

Reply:
I added tests under `TestArithmeticIntDivide `, which covers cases of not nil err. Also switch case seems wrong in `TestArithmeticDivide `, as `ast.Div` applies only for `Real` and `Decimal` type.